Packs/PrismaCloud/Layouts/layout-details-GCP_Kubernetes_Engine_Misconfiguration.json

```diff
@@ -0,0 +1,698 @@
+							"id": "18654c40-a9d9-11e9-a91e-57df9227e86a",
+							"index": 5,
+							"listId": "9c8d74a0-93c4-11e9-9dd5-3b4f6f9f1bae",
+							"startCol": 0
+						},
+						{
+							"endCol": 2,
+							"fieldId": "resourceapiname",
+							"height": 24,
+							"id": "0a428600-a9d9-11e9-a91e-57df9227e86a",
+							"index": 6,
+							"startCol": 0
+						},
+						{
+							"dropEffect": "move",
+							"endCol": 2,
+							"fieldId": "resourcecloudtype",
+							"height": 24,
+							"id": "cc430c50-93c4-11e9-9dd5-3b4f6f9f1bae",
+							"index": 7,
+							"listId": "9c8d74a0-93c4-11e9-9dd5-3b4f6f9f1bae",
+							"startCol": 0
+						}
+					],
+					"maxW": 3,
+					"minH": 1,
+					"minW": 1,
+					"moved": false,
+					"name": "Violating Resource",
+					"static": false,
+					"w": 1,
+					"x": 1,
+					"y": 0
+				},
+				{
+					"description": "Policy compliance-related information.",
+					"displayType": "CARD",
+					"h": 3,
+					"hideItemTitleOnlyOne": true,
+					"hideName": false,
+					"i": "fyjdlofdhd-98793800-a9d5-11e9-a91e-57df9227e86a",
+					"isVisible": true,
+					"items": [
+						{
+							"dropEffect": "move",
+							"endCol": 6,
+							"fieldId": "compliancemetadata",
+							"height": 110,
+							"id": "bea01080-a9d5-11e9-a91e-57df9227e86a",
+							"index": 1,
+							"listId": "fyjdlofdhd-98793800-a9d5-11e9-a91e-57df9227e86a",
+							"startCol": 0
+						}
+					],
+					"maxW": 3,
+					"minH": 1,
+					"minW": 1,
+					"moved": false,
+					"name": "Policy Compliance Metadata",
+					"static": false,
+					"w": 3,
+					"x": 0,
+					"y": 6
+				},
+				{
+					"description": "Alert Rules created using the policy.",
+					"displayType": "CARD",
+					"h": 3,
+					"hideItemTitleOnlyOne": true,
+					"hideName": false,
+					"i": "fyjdlofdhd-1adf4900-f761-11e9-8abd-b54c9337d32a",
+					"items": [
+						{
+							"dropEffect": "move",
+							"endCol": 6,
+							"fieldId": "prismacloudrules",
+							"height": 110,
+							"id": "aabb18d0-f8b8-11e9-a470-b7537b797c27",
+							"index": 0,
+							"listId": "fyjdlofdhd-1adf4900-f761-11e9-8abd-b54c9337d32a",
+							"startCol": 0
+						}
+					],
+					"maxW": 3,
+					"minH": 1,
+					"minW": 1,
+					"moved": false,
+					"name": "Alert Rules",
+					"static": false,
+					"w": 3,
+					"x": 0,
+					"y": 3
+				}
+			],
+			"type": "custom"
+		},
+		{
+			"id": "warRoom",
+			"name": "War Room",
+			"type": "warRoom"
+		},
+		{
+			"id": "workPlan",
+			"name": "Work Plan",
+			"type": "workPlan"
+		},
+		{
+			"id": "evidenceBoard",
+			"name": "Evidence Board",
+			"type": "evidenceBoard"
+		},
+		{
+			"id": "relatedIncidents",
+			"name": "Related Incidents",
+			"type": "relatedIncidents"
+		},
+		{
+			"id": "canvas",
+			"name": "Canvas",
+			"type": "canvas"
+		}
+	],
+	"typeId": "GCP Kubernetes Engine Misconfiguration",
+	"version": -1
+	},
+	"typeId": "GCP Kubernetes Engine Misconfiguration",
+	"version": -1,
+	"toVersion": "5.9.9"
```

Comment:
why you limit the version to 5.9.9 here?

Reply:
I didn't change the value of `toVersion`.  The value `5.9.9` is from the Demisto export.

Reply:
is this to support versions prior to 6.0?